src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -66,6 +69,9 @@ const (
 	maxSegmentArrayPooledLength = 32
 	// Any pooled error slices that grow beyond this capcity will be thrown away.
 	writeBatchPooledReqPoolMaxErrorsSliceSize = 4096
+
+	fetchTaggedSpanID = "rpc_fetch_tagged"
+	querySpanID       = "rpc_query"
```

Comment:
another point (this one I'm not as sure of) - how would you feel about moving all these constants as exported in a separate package (say `src/dbnode/tracepoint`). Would help discoverability. And also provide a nice single place to document the convention of naming these things.

Reply:
If we go with the tags approach, i'd say leave it in the individual files.